code/__HELPERS/game.dm

```diff
@@ -481,3 +481,41 @@ proc/pollCandidates(var/Question, var/be_special_type, var/antag_age_check = 0,
 			candidates.Remove(G)
 
 	return candidates
+
+// Split block - A version of block() that splits it into multiple 32x32 block() calls to allow CHECK_TICK and SCHECK to work
+/datum/sub_turf_block
+	var/x1
+	var/y1
+	var/z1
+	var/x2
+	var/y2
+	var/z2
+
+/datum/sub_turf_block/New(x1, y1, z1, x2, y2, z2)
+	src.x1 = x1
+	src.y1 = y1
+	src.z1 = z1
+	src.x2 = x2
+	src.y2 = y2
+	src.z2 = z2

```

Comment:
Goddammit give your arguments names that aren't the same as the datum's variables.